### ISSUE TITLE
arlington checker complains about null entries in the Parenttree

### ISSUE THREAD

**Issue by u-fischer** [veraPDF/veraPDF-library#1355]
There are PDF producer which create a ParentTree where unused slots without a related MCID number are represented by a `null` object:

~~~~
/Nums [ 0 [ null null null null null null 89 0 R null null 90 0 R null null null null 91 0 R]
             ....
          ]
~~~~

The arlington checker complains a lot about this:

~~~~
root/Root[0](ACatalog 2 0 obj){Root}/StructTreeRoot[0](AStructTreeRoot 39 0 obj){StructTreeRoot}/ParentTree[0]{ParentTree}/Entries[0]/Entry[0]/Entries[0]{0}
Entry 0 in ArrayOfStructElem is not of type Dictionary

root/Root[0](ACatalog 2 0 obj){Root}/StructTreeRoot[0](AStructTreeRoot 39 0 obj){StructTreeRoot}/ParentTree[0]{ParentTree}/Entries[0]/Entry[0]/Entries[1]{1}
Entry 1 in ArrayOfStructElem is not of type Dictionary
~~~~

An example document is for example
https://www.axes4.com/_Resources/Persistent/d/6/9/b/d69b8c8400099b8651a599e7aa520f0a52a22035/axesWord_PDF-A_DE_Info-Sheet_V22-09-09.pdf 
(I can't show a LaTeX examples as we avoid gaps in the MCID numbers)

Imho such null entries are allowed.

**Comment by bdoubrov** [comment]
Currently Arlington model doesn't allow `null` in arrays of structure elements. See the related Arlington issue

**Comment by MaximPlusov** [comment]
Fixed in the latest dev build 1.25.97.
